dockers/llm.vdb.service/common.py

```diff
@@ -0,0 +1,68 @@
+import json
+import os
+
+from langchain.text_splitter import RecursiveCharacterTextSplitter
+from langchain_community.vectorstores import FAISS
+from langchain_huggingface import HuggingFaceEmbeddings
+
+
+def load_jsonl_files_from_directory(directory):
+    data = []
+    # Loop through all files in the directory
+    for filename in os.listdir(directory):
+        if filename.endswith(".jsonl") or filename.endswith(".json"):
+            file_path = os.path.join(directory, filename)
+            # Open and read each jsonl file
+            with open(file_path, "r") as file:
+                for line in file:
+                    # Parse each JSON object in the file
+                    data.append(json.loads(line.strip()))
+    return data
+
+
+def get_documents_with_metadata(data):
+    texts = [doc["text"] for doc in data]
+    metadatas = [doc["metadata"] for doc in data]
+    return texts, metadatas
+
+
+def chunk_documents_with_metadata(data, chunk_size=1000, chunk_overlap=200):
```

Comment:
Could we use the default consts defined in createvectordb.py here instead so we will have one default? Perhaps these consts can be moved to common.py and then used here:
`EMBEDDING_CHUNK_SIZE_DEFAULT`
`EMBEDDING_CHUNK_OVERLAP_DEFAULT`

Reply:
changed